### ISSUE TITLE
keras

### ISSUE THREAD

**Issue by sungyoon-lee** [sungyoon-lee/to-do#73]
https://keras.io/

**Comment by sungyoon-lee** [comment]
symbolic

**Comment by sungyoon-lee** [comment]
K.batch_dot

**Comment by sungyoon-lee** [comment]
flags

**Comment by sungyoon-lee** [comment]
np.max axis
axis=0 means along column
If this is a tuple, calculate maximum along multiple axes

**Comment by sungyoon-lee** [comment]
K.equal
elementwise equal

**Comment by sungyoon-lee** [comment]
```python
import tensorflow as tf

FLAGS = tf.app.flags.FLAGS
FLAGS.learning_rate = 0.02
FLAGS.name = 'test'
```
print(FLAGS.learning_rate)
print(FLAGS.name)

**Comment by sungyoon-lee** [comment]
```python
flags = tf.app.flags
FLAGS = flags.FLAGS
flags.DEFINE_float('learning_rate', 0.01, 'Initial learning rate.')
```

**Comment by sungyoon-lee** [comment]
```python
from keras.preprocessing.image import ImageDataGenerator
def data_gen_mnist(X_train):
    datagen = ImageDataGenerator()

    datagen.fit(X_train)
    return datagen
```
